src/main/java/net/sf/jabref/openoffice/CitationManager.java

```diff
@@ -140,20 +140,20 @@ public void showDialog() {
     static class CitEntry implements Comparable<CitEntry> {
 
         private final String refMarkName;
-        private String pageInfo;
+        private Optional<String> pageInfo;
         private final String context;
-        private final String origPageInfo;
+        private final Optional<String> origPageInfo;
 
 
-        public CitEntry(String refMarkName, String context, String pageInfo) {
+        public CitEntry(String refMarkName, String context, Optional<String> optional) {
```

Comment:
OK, the thing is this:

``` java
            list.add(new CitEntry(name,
                    "<html>..." + ooBase.getCitationContext(nameAccess, name, 30, 30, true) + "...</html>",
                    ooBase.getCustomProperty(name)));
```

since getCustomProperty now returns an Optional it makes sense to have an Optional as a parameter, if nothing else for brevity.